dist/gcc-compatible/configure

```diff
@@ -245,12 +250,21 @@ if [ ! -z "$build_target" ]; then
     CC="$TOOLCHAIN/bin/aarch64-none-linux-gnu-gcc"
     echo "CC=$CC" >> Makefile.config
     # Using GCC here doesn't require setting anything else.
+  elif [[ "$build_target" == "ia32" ]]; then
```

Comment:
the `ia32` name seems a little unfamiliar; I remember `i686` and `i386` being more common in, say, configure triples (e.g. i686-linux-gnu) or in `uname -m` return values (see https://stackoverflow.com/questions/45125516/possible-values-for-uname-m) for some information -- should we settle on i686 as the one possible value to indicate 32-bit x86?

Reply:
`ia32` is the generic name for Intel 32-bit (https://en.wikipedia.org/wiki/IA-32). The issue is that there are so many different architectures that are 32-bit and they all have another architecture name (like the ones I match `detect_x86` and more).
I don't have strong opinions on the names we use here. The most generic way would be to not actually specify the target here but use a separate argument like `-m32` to indicate a 32-bit build. This would mimic the compiler flag we set anyway.

Reply:
ok no need to change then -- thanks for the background, `ia32` is good!

Reply:
agreed that there is confusion between i386/i686 which are revisions of the architecture